http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -383,25 +365,15 @@ class DataprocInterpreter[F[_]: Async: Parallel: ContextShift: Logger](
       }
     }
 
-  def createClusterIamRoles(googleProject: GoogleProject, serviceAccountInfo: ServiceAccountInfo): F[Unit] =
+  def createClusterIamRoles(googleProject: GoogleProject, serviceAccountInfo: WorkbenchEmail): F[Unit] =
     updateClusterIamRoles(googleProject, serviceAccountInfo, createCluster = true)
 
-  def removeClusterIamRoles(googleProject: GoogleProject, serviceAccountInfo: ServiceAccountInfo): F[Unit] =
+  def removeClusterIamRoles(googleProject: GoogleProject, serviceAccountInfo: WorkbenchEmail): F[Unit] =
     updateClusterIamRoles(googleProject, serviceAccountInfo, createCluster = false)
 
   def generateServiceAccountKey(googleProject: GoogleProject,
```

Comment:
I think we can remove `generateServiceAccountKey` as well?

Reply:
cool..I wasn't sure and left the question in description :)